src/Infrastructure/LeanCode.AuditLogs/StubAuditLogStorage.cs

```diff
@@ -0,0 +1,24 @@
+using Serilog;
+
+namespace LeanCode.AuditLogs;
+
+public class StubAuditLogStorage : IAuditLogStorage
```

Comment:
I wonder if we should ever provide this implementation - it's somewhat useless for real projects.